graphdatascience/graph_data_science.py

```diff
@@ -74,7 +75,7 @@ def __init__(
 
             driver = GraphDatabase.driver(endpoint, auth=auth, **self._config)
 
-            self._query_runner = Neo4jQueryRunner(driver, auto_close=True)
+            self._query_runner = Neo4jQueryRunner(driver, auto_close=True, bookmarks=bookmarks)
```

Comment:
Did you know about https://neo4j.com/docs/api/python-driver/current/api.html#neo4j.api.BookmarkManager ? 

The drivers team recommended using it :)

Reply:
I think passing an Optional BookMarkManger could make more sense

Reply:
also they note the potential performance impact, so keeping it optional makes sense.
(https://neo4j.com/docs/api/python-driver/current/api.html#bookmark-manager-ref)

Reply:
I've just read https://neo4j.com/docs/python-manual/current/bookmarks/ To implement a bookmarkmanager seems a bit overkill since we have exactly the bookmark support we need already in the session API?

Reply:
i think you are right.
Also thinking further, I imagine we dont need to pass the bookmarks to the server afterall.
As any query run inside the session will be after the bookmark. So internal transactions created f.i. during projection should still be after the bookmark here.